api/src/main/java/org/pmiops/workbench/disks/DiskAdminService.java

```diff
@@ -146,6 +151,12 @@ public void checkPersistentDisks(List<Disk> disks) {
     }
   }
 
+  private boolean shouldNotify(int daysUnused) {
```

Comment:
don't send for newly idle disks (0 days)

do send the first time the disk age crosses the INACTIVE_DISK_NOTIFY_THRESHOLDS_DAYS (14 days) as well as every multiple of INACTIVE_DISK_NOTIFY_PERIOD_DAYS (30d, 60d, etc)

Reply:
I think you could do: `daysUnused > 0 && daysUnused % INACTIVE_DISK_NOTIFY_PERIOD_DAYS == 0`

Reply:
that misses the `INACTIVE_DISK_NOTIFY_THRESHOLDS_DAYS` condition

Reply:
Oh never mind, I missed that there are two different constants being used there.